turnerlabs/yt_push
# DRACO-3597 Description field for YouTube videos

Finished the acceptance criteria on DRACO-3597 with the addition of a description field.

To test:

* Create a new YouTube media type using the Drush command: `drush cay`.
* Authenticate that media type.
* Create a new YouTube media with a description and check that the description was reflected in the uploaded video.
